File: Python3/프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    k = 0
    t = 0
    answer = [k,t] 
    for i in num_list:
        if i%2 == 0:
            k +=1
        else:
            t +=1
        answer[0] = k 
        answer[1] = t
    return answer

File: Java/프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int result1 = 0;
        int result2 = 0;
        for(int i=0; i<num_list.length; i++){
            if(num_list[i]%2==0) result1++;
            else result2++;
        }
        int[] answer = new int[2];
        answer[0] = result1;
        answer[1] = result2;
        return answer;
    }
}